DataFormats/Detectors/TRD/src/RawData.cxx

```diff
@@ -292,25 +183,25 @@ DigitMCMADCMask constructBlankADCMask()
   return mask;
 }
 
-void printTrackletHCHeader(o2::trd::TrackletHCHeader& halfchamber)
+void printTrackletHCHeader(const o2::trd::TrackletHCHeader& halfchamber)
```

Comment:
There were many many duplicates and I tried to remove most of them. Probably I have missed some. It might make sense to have both an `operator<<` and a `print()` method. With the streamer you can assemble a log message without line breaks. I am not using the `operator<<` though so for me it would be fine to remove this

Reply:
the duplicates were to make a one liner and a more verbose version as far as i remember.